internal/stack/logs.go

```diff
@@ -12,17 +12,27 @@ import (
 	"github.com/elastic/elastic-package/internal/compose"
 	"github.com/elastic/elastic-package/internal/docker"
 	"github.com/elastic/elastic-package/internal/install"
+	"github.com/elastic/elastic-package/internal/profile"
 )
 
-func dockerComposeLogs(serviceName string, snapshotFile string) ([]byte, error) {
+func dockerComposeLogs(serviceName string, profile *profile.Profile) ([]byte, error) {
```

Comment:
Use `profile` as parameter instead of using `snapshotFile` as parameter.
Having the profile as parameter, allows us:
- obtain the snapshot file path
- get the needed Environment variables from the Profile for the `docker-compose logs` command